install/kubernetes/cilium/Chart.yaml

```diff
@@ -1,9 +1,77 @@
 apiVersion: v2
 name: cilium
+displayName: Cilium
 home: https://cilium.io/
 version: 1.9.90
 appVersion: 1.9.90
-description: Helm chart for Cilium
+kubeVersion: ">= 1.12.0-0"
+icon: https://raw.githubusercontent.com/cilium/cilium/master/Documentation/images/logo-solo.svg
```

Comment:
@joestringer I assume we should make this logo dependent on the branch we are? i.e. if we backport this to 1.9 we need to change `cilium/master` to `cilium/v1.9`?

Reply:
Addressed with the top commit.